runtime/lua/vim/diagnostic.lua

```diff
@@ -1,6 +1,8 @@
 local api, if_nil = vim.api, vim.F.if_nil
 
-local M = {}
+local M = {
+  _qf_id = nil,
+}
```

Comment:
this could be a local variable in the module, no need to quasi-expose it in the public interface.

```suggestion
local M = {}

_qf_id = nil
```

Reply:
Done!